public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -131,6 +136,11 @@ export const RuleActionsButtons = ({ compact, showViewButton, showCopyLinkButton
         }}
       />
       {deleteModal}
+      {isGrafanaRulerRule(rule.rulerRule) && showSilenceDrawer && (
+        <AlertmanagerProvider accessType="instance">
```

Comment:
I guess the `accessType=instance` might be no longer valid here. It assumes you need the `instance` permission which is not true in this context
I think it should be silence for GMA and instance for external AMs 🤔

Reply:
(As per Slack, will address this in the RBAC portion)